core/src/test/java/io/questdb/griffin/InsertTest.java

```diff
@@ -587,11 +704,13 @@ public void testInsertWithoutNominatedTimestampAndTypeDoesNotMatch() throws Exce
     }
 
     @Test
+    @Ignore
     public void testInsertWrongTypeConstant() throws Exception {
         assertMemoryLeak(() -> {
             compiler.compile("create table test (a timestamp)", sqlExecutionContext);
             try {
                 compiler.compile("insert into test values ('2013')", sqlExecutionContext);
+                Assert.fail();
```

Comment:
This test was passing incorrectly beforehand so probably needs looking at. There was no exception being thrown. Have added this assertion to fail the test if no exception thrown and ignored the test for now.